fault-readme.md

```diff
@@ -125,6 +126,18 @@ If the trigger_address is defined as "-1", the trigger address is set to one ins
 
 Currently, only addresses in decimal representation are accepted. 
 
+###### fault_exclude
+
+For the wildcard mode there is the possibility to exclude certain instructions.
+The fault_exclude address ranges are specified by start and end.
+If the instruction pointer is within this range it will not be added to the fault list.
```

Comment:
Instruction pointer might be confusing.

Perhaps: "No faults are generated for addresses within this range."
and "Hence, a fault address equal to the end value is not excluded."